examples/first-to-market-core.md

```diff
@@ -0,0 +1,7 @@
+# First-to-Market Core
+
+![First to  market core](/resources/first-to-market-core.png)
+
+A First-to-market Core provides an advantage to the organisation which develops the capability first. But since the complexity of the feature is low, the competition will soon catch up and it will not be the core domain for a long period of time.
+
```

Comment:
A very cool description of the problem but I think it is not really an example, rather an explanation. What about adding a real-life story like 
```suggestion
Let say, you have a product like Meetup but yo have some great ideas about how to beat them in the long run. You know that your core business is scheduling and managing events but this is only your home-work. Soon after this is in place you will focus on the unique selling point for your product, like supporting different remote working tools (?).
```

What do you think? Maybe you have a better example, it is not so easy to come up with the 5-billion product idea in 2 minutes :D

Reply:
Yes, this is very important to examples. Thanks.